garage.com/src/main/java/ConvertibleCar.java

```diff
@@ -1,6 +1,30 @@
 public class ConvertibleCar extends Car implements ICar {
 
-	public ConvertibleCar(String model, IEngine engine, IIgnition ignitionSystem) {
-		super(model, engine, ignitionSystem);
+	public ConvertibleCar(String model, IEngine engine, IIgnition ignitionSystem, String lowerTop, String raiseTop) {
```

Comment:
You cannot extend Car with lowerTop and raiseTop functionality because Car does not have this capability.

Reply:
Resolved